Assets/Scripts/Interaction System/Interactable.cs

```diff
@@ -10,7 +10,7 @@ public class Interactable : MonoBehaviour
         public UnityEvent<Character> onInteraction;
 
         private bool _canBeInteracted;
-        private Character _interactor;
+        public Character _interactor;
```

Comment:
This only depends on the interactable itself. no need to make it public

Reply:
done